gui/create-item.lua

```diff
@@ -184,11 +211,183 @@ local function getCreatureRaceAndCaste(caste)
  return df.global.world.raws.creatures.list_creature[caste.index],df.global.world.raws.creatures.list_caste[caste.index]
 end
 
+function createCorpsePiece(creator, bodypart, partlayer, creatureID, casteID, generic, quality) -- this part was written by four rabbits in a trenchcoat (ppaawwll)
+ -- (partlayer is also used to determine the material if we're spawning a "generic" body part (i'm just lazy lol))
+ quality = math.max(0, math.min(5, quality - 1))
+ local creat = tonumber(creatureID)
```

Comment:
Since you never use `creatureID` after this, you could just normalize it with:
```suggestion
 creatureId = tonumber(creatureID)
```